README.md

```diff
@@ -2193,6 +2188,25 @@ _Unofficial libraries for package and dependency management._
 
 **[⬆ back to top](#contents)**
 
+## Reflection
+
+- [autowire](https://github.com/tiendc/autowire) - Dependency injection using Generics and reflection.
+- [boot-go](http://github.com/boot-go/boot) - Component-based development with dependency injection using reflections for Go developers.
+- [copy](https://github.com/gotidy/copy) - Package for fast copying structs of different types.
+- [Deepcopier](https://github.com/ulule/deepcopier) - Simple struct copying for Go.
+- [dig](https://github.com/uber-go/dig) - A reflection based dependency injection toolkit for Go.
+- [go-deepcopy](https://github.com/tiendc/go-deepcopy) - Fast deep copy library.
+- [go-fixedwidth](https://github.com/ianlopshire/go-fixedwidth) - Fixed-width text formatting (encoder/decoder with reflection).
```

Comment:
`go-fixedwidth` fits more in **[Parsers/Encoders/Decoders](https://github.com/avelino/awesome-go#parsersencodersdecoders)**